anaDACScan.py

```diff
@@ -156,9 +156,9 @@
         for line in open(args.calFileList):
             if line[0] == "#":
                 continue
-            line = line.strip(' ').strip('\n')
-            first = line.split(' ')[0].strip(' ')
-            second = line.split(' ')[1].strip(' ')
+            line = line.strip()
+            first = line.split()[0]
+            second = line.split()[1]
```

Comment:
While it won't matter for this application you should get in the habit of not calling subsequent actions like this.  If `line` is a large string you'll be spending time parsing it twice.  It would be better to parse it once:

```python
line = line.strip().split()
first = line[0]
second = line[1]
```

Fine for now; but pay closer attention to this in the future.